src/applications/check-in/utils/pilotFeatures/index.js

```diff
@@ -0,0 +1,82 @@
+import wilkesBarre from './travelClaim/wilkesBarre.json';
```

Comment:
We could end up with naming collisions for other features. Maybe something like `travelClaimStationsWilkesBarre`